script.js

```diff
@@ -0,0 +1,32 @@
+const rock = "rock";
+const paper = "paper";
+const scissors = "scissors";
+const compVal = ['rock','paper','scissors'];
```

Comment:
This constant name in really strange, it doesn't really say what the constant contains, a good name would be something like `allShapes`, or something like that